tokio/src/io/util/repeat.rs

```diff
@@ -56,10 +58,7 @@ impl AsyncRead for Repeat {
     ) -> Poll<io::Result<()>> {
         ready!(crate::trace::trace_leaf(cx));
         ready!(poll_proceed_and_make_progress(cx));
-        // TODO: could be faster, but should we unsafe it?
-        while buf.remaining() != 0 {
-            buf.put_slice(&[self.byte]);
-        }
+        buf.put_bytes(self.byte, buf.remaining());
```

Comment:
You need to increase the dependency on the `bytes` crate to require at least version 1.1.0 for this method to be available.